integration/use-cases/create-private-registry.js

```diff
@@ -16,120 +16,120 @@ test("Creates a private registry", async () => {
 
   const randomNumber = Math.floor(Math.random() * Math.floor(100));
   const repoName = "my-repo-" + randomNumber;
-  await page.type("#kubeapps-repo-name", repoName);
+  await page.type("cds-form-group > cds-input:nth-child(1) > input", repoName);
 
   await page.type(
-    "#kubeapps-repo-url",
```

Comment:
the component ID of the inputs is automatically generated on the fly so we cannot use these fixed IDs anymore